integration-cli/docker_cli_build_test.go

```diff
@@ -2293,6 +2293,29 @@ func TestBuildExposeOrder(t *testing.T) {
 	logDone("build - expose order")
 }
 
+func TestBuildExposeHostPort(t *testing.T) {
```

Comment:
When run `docker build .`,  I can see the warning added by this patch "Dockerfile EXPOSE (ip:hostPort:containerPort) ..."
But inside testcases, got nothing from buildImageWithOut.